docs/content/index.md

```diff
@@ -11,7 +11,7 @@
 | [Conflux Expansion](conflux_expansion.md) | [Conflux Faction](../towns/conflux.md)<br>Monoliths |
 | [Cove Expansion](cove_expansion.md) | [Cove Faction](../towns/cove.md)<br>[Sea Tiles](../keywords/sea_tile.md) |
 | [Battlefield Expansion](battlefield_expansion.md) | Hex Battlefield With Obstacles<br>Adventure<br>Positive Morale Cards<br>Negative Morale Cards |
-| [Naval Battles Expansion](naval_battles_expansion.md) | 2 Neoprene Battlefield mats<br>Creature Banks |
+| [Naval Battles Expansion](naval_battles_expansion.md) | 2 Neoprene Battlefield mats<br>Creature Banks<br>Neutral Units<br>Tokens |
```

Comment:
No granularity for those tokens? Those are empowered ability ones and stack tokens?